catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -118,6 +123,8 @@ public void init() {
 
     post("/cql/transform/:transformerId", cqlTransformHandler, GSON::toJson);
 
+    post("/cql/validate/:validatorId", cqlValidateHandler, GSON::toJson);
```

Comment:
❓ Not sure if this is supposed to be a REST endpoint if it is we should get away from using verbs as part of our resource naming conventions
https://www.restapitutorial.com/lessons/restfulresourcenaming.html
/cql/validator/:validatorId